core/field_checkbox.js

```diff
@@ -127,11 +145,20 @@ Blockly.FieldCheckbox.prototype.initView = function() {
   this.textElement_.setAttribute('y', Blockly.FieldCheckbox.CHECK_Y_OFFSET);
   Blockly.utils.dom.addClass(this.textElement_, 'blocklyCheckbox');
 
-  var textNode = document.createTextNode(Blockly.FieldCheckbox.CHECK_CHAR);
-  this.textElement_.appendChild(textNode);
+  this.textContent_.nodeValue =
+      this.checkChar_ || Blockly.FieldCheckbox.CHECK_CHAR;
   this.textElement_.style.display = this.value_ ? 'block' : 'none';
 };
 
+/**
+ * Set the character used for the check mark.
+ * @param {string} character The character to use for the checkmark.
```

Comment:
@package for method visibility.

Reply:
It's a developer hook, so I think it should be public. Should it be tagged @public?

Reply:
Definitely.

Reply:
Oh yes, so leave as is.